tetraka/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# pull official base image
+FROM node:14.9.0-alpine
+
+# set working directory
+WORKDIR /app
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install app dependencies
+COPY package.json ./
+COPY package-lock.json ./
+RUN npm install --silent
+RUN npm install react-scripts@3.4.1 -g --silent
```

Comment:
do we need to specify this dependency specifically? why not bundled into package.json?

Reply:
Already built in.